### ISSUE TITLE
Video playback is stopped at the end of the video when opened from Jellyfin

### ISSUE THREAD

**Issue by BinaryWarlock** [abdallahmehiz/mpvKt#177]
- Play video in Jellyfin, open with mpvKt 0.1.6
- Video opens stopped at the end of the video (e.g. 32:00 of a 32:00 minute video)

Have to seek to the beginning for some reason to actually play it. Regular mpv-android opens from the beginning and plays normally.

**Comment by abdallahmehiz** [comment]
mpvKt is probably saving the playback states of other videos from jellyfin and reusing it in new videos.
Try disabling the save playback state in the player settings.

**Comment by BinaryWarlock** [comment]
You're right, it uses the position of the file last played, and turning off "Save playback on quit" fixes it, although not ideal.
